package.json

```diff
@@ -21,7 +22,9 @@
     "eject": "react-scripts eject",
     "lint": "eslint --max-warnings 0 --ext .tsx,.ts ./src",
     "lint:fix": "eslint --fix --ext .tsx,.ts,.js,.jsx ./src",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "generate:api": "restful-react import --file schema.json --output ./src/apiClient.tsx",
+    "postgenerate:api": "eslint --fix ./src/apiClient.tsx"
```

Comment:
Hmm, my take is we should probably just add `apiClient.tsx` to an `.eslintignore` file - it's a generated file, no point forcing generated code conform to our lint standards.

Reply:
It is somewhat useful to have it formatted consistently though as a way to make reading through the file easier. The library itself does suggest this post step.

Reply:
> It is somewhat useful to have it formatted consistently though as a way to make reading through the file easier. The library itself does suggest this post step.

Fair enough, keeps the codebase from becoming inconsistent :)